SU-HSDO/suhumsci
# SHS-5003: Sort paragraphs in Paragraphs Browser by weight

# READY FOR REVIEW

## Summary
Add patch that sorts configured paragraph types by weight on the configuration form
Merge only after #1305 is deployed

## Need Review By (Date)
_['10/30', 'asap', etc.]_

## Urgency
_['low', 'medium', 'high', etc.]_

## Steps to Test
2. Add a paragraph to an entityreference revision field (Full Width Region or Main Region) where paragraph options were configured by exclusion,
2. Check the paragraphs browser, I see the paragraph options sorted by weight

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
